pdf/package.json

```diff
@@ -32,17 +32,18 @@
   },
   "devDependencies": {
     "@babel/eslint-parser": "7.24.1",
+    "@intlify/core": "^9.10.2",
     "@rushstack/eslint-patch": "1.8.0",
     "@vitejs/plugin-vue": "5.0.4",
     "@vitest/coverage-v8": "1.4.0",
     "@vue/babel-preset-app": "5.0.8",
     "@vue/compiler-dom": "3.4.21",
     "@vue/compiler-sfc": "3.4.21",
     "@vue/eslint-config-prettier": "9.0.0",
-    "@vue/test-utils": "2.4.4",
     "@vue/runtime-dom": "3.4.21",
     "@vue/server-renderer": "3.4.21",
     "@vue/shared": "3.4.21",
+    "@vue/test-utils": "2.4.4",
```

Comment:
why did you move @vue/test-utils down?

Reply:
It was automatically sorted this way when I installed the new package